src/daml/Daml/Trading/Account/Account.daml

```diff
@@ -81,6 +90,8 @@ template Account
     address: Text 
     optCashAccount: Optional (ContractId Account.F, ContractId Holding.F, AccountKey) 
     optNftAccount: Optional (ContractId Account.F, ContractId Holding.F, AccountKey) 
+    rewardPoint : Int
+    badges : [BadgeType]
```

Comment:
We should use Set instead of array, since Set cannot contain duplicate elements
Also, we should create a template for Badge instead of simple data because we may need to add more properties to Badge in the future, or count the number of Badge issued, or to have some choices on it, for example like redeeming Badge for money

Reply:
I created Badge template but type of badges, I set it is ContractId Badge.Badge type